dbms/src/Debug/MockRaftStoreProxy.cpp

```diff
@@ -180,9 +180,10 @@ static RaftstoreVer fn_get_cluster_raftstore_version(RaftStoreProxyPtr ptr, uint
 }
 
 // Must call `RustGcHelper` to gc the returned pointer in the end.
-static RustStrWithView fn_get_config_json(RaftStoreProxyPtr, uint64_t)
+static RustStrWithView fn_get_config_json(RaftStoreProxyPtr ptr, uint64_t)
 {
-    auto * s = new std::string(R"({"raftstore":{"snap-handle-pool-size":4}})");
+    auto & x = as_ref(ptr);
+    auto * s = new std::string(x.proxy_config_string);
```

Comment:
why we need to create the `x = as_ref(ptr)`? Can we directly use `auto * s = new std::string(ptr->proxy_config_string);`?

Reply:
There is no `proxy_config_string` in `RaftStoreProxyPtr`.
You can consider `RaftStoreProxyPtr` as a `void*` passed between proxy and tiflash by ffi.

Reply:
ok, I see `as_ref` is a `reinterpret_cast`